runtimevar/constantvar/constantvar.go

```diff
@@ -26,13 +26,14 @@ import (
 	"errors"
 	"time"
 
+	"gocloud.dev/gcerrors"
 	"gocloud.dev/runtimevar"
 	"gocloud.dev/runtimevar/driver"
 )
 
-// ErrNotExist is a sentinel error that can be used with NewError to return an
+// errNotExist is a sentinel error that can be used with NewError to return an
```

Comment:
Comment is stale; maybe just delete it as it's private now.

Reply:
Done.